Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -1592,29 +1593,34 @@ private final class TokenStreamCreator: SyntaxVisitor {
           // comment if the user places one here but the comment is otherwise adjacent to a text
           // token.
           appendToken(.break(.same, size: 0))
+          isStartOfFile = false
         }
 
       case .docLineComment(let text):
         appendToken(.comment(Comment(kind: .docLine, text: text), wasEndOfLine: false))
         appendToken(.newline)
+        isStartOfFile = false
 
       case .docBlockComment(let text):
         appendToken(.comment(Comment(kind: .docBlock, text: text), wasEndOfLine: false))
         appendToken(.newline)
+        isStartOfFile = false
 
       case .newlines(let count), .carriageReturns(let count), .carriageReturnLineFeeds(let count):
-        if config.respectsExistingLineBreaks && isDiscretionaryNewlineAllowed(before: token) {
-          appendToken(.newlines(count, discretionary: true))
-        }
-        else {
-          // Even if discretionary line breaks are not being respected, we still respect multiple
-          // line breaks in order to keep blank separator lines that the user might want.
-          // TODO: It would be nice to restrict this to only allow multiple lines between statements
-          // and declarations; as currently implemented, multiple newlines will locally override the
-          // configuration setting.
-          if count > 1 {
+        if !isStartOfFile {
```

Comment:
Nit: `guard !isStartOfFile else { break }` to avoid the extra indentation.

Reply:
Done.